cli/ballerina-cli/src/main/java/io/ballerina/cli/task/CreateExecutableTask.java

```diff
@@ -95,6 +101,31 @@ public void execute(Project project) {
                 start = System.currentTimeMillis();
             }
             jBallerinaBackend.emit(JBallerinaBackend.OutputType.EXEC, executablePath);
+
+            if (enableNativeImage) {
+                // Create nativeConfig directory
+                if (!Files.exists(target.path().resolve("native"))) {
+                    Files.createDirectory(target.path().resolve("native"));
+                }
+
+                String nativeImageDir = "target" + File.separator + "native" + File.separator;
+                String nativeImageName = project.currentPackage().packageName().toString();
+                String[] command = {"native-image", "-jar", executablePath.toString(),
+                        "." + File.separator + nativeImageDir + nativeImageName};
```

Comment:
Don't we need to add more flags to support ballerina in all the cases? @TharmiganK  @warunalakshitha

Reply:
Can we add `--no-fallback` as the default option? In that case if the native image tool cannot create a standalone, then it will give an error
But [native-image tool has not done that due to some legal reasons](https://github.com/oracle/graal/issues/2648), not sure what are those.

In terms of other flags, they will be automatically added through ballerina runtime and standard library modules

Reply:
Added `--no-fallback` as a default option